src/frontend/js/custom-leaflet.js

```diff
@@ -95,7 +95,7 @@ function main_map_init (map, options) {
     // Create bike router using mapbox
     var bikerouter = L.Routing.mapbox('pk.eyJ1IjoicmphY3F1ZW1pbiIsImEiOiJjaXAxaWpxdGkwMm5ydGhtNG84eGdjbGthIn0.TdwCw6vhAJdgxzH0JBp6iA');
     var defaultVehicle = $('#route_map').data('selected-vehicle');
-    bikerouter.options.profile = 'mapbox.' + defaultVehicle;
+    bikerouter.options.profile = 'mapbox/' + defaultVehicle;
```

Comment:
Yes. Routing profiles in the Mapbox Directions API v4 have a period in them, whereas in v5 they have a slash.